src/components/Rockets.css

```diff
@@ -11,3 +11,28 @@
 .rocket-info {
   width: 70vw;
 }
+
+.reserved-badge {
+  color: azure;
+  font-size: 0.8rem;
+  padding: 3px;
+  background-color: rgb(15, 161, 161);
+  border: 2px solid rgb(185, 185, 35);
+  border-radius: 10px;
+}
+
+.btn {
+  font-size: 1.2rem;
+  padding: 8px;
+  border-radius: 15px;
+}
+
+.rocket-reserve.btn {
+  color: azure;
+  background-color: orangered;
+}
+
+.rocket-cancel.btn {
+  color: orangered;
+  background-color: aquamarine;
+}
```

Comment:
- [ ] The design should be the same. Kindly update it. [Requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#project-requirements)

![image](https://user-images.githubusercontent.com/66992901/182790208-a678a257-6897-4833-b833-4b8ca723f3cc.png)


___
### ORIGINAL DESIGN
![image](https://github.com/microverseinc/curriculum-react-redux/raw/main/group-project/images/space-travelers__rockets.png)